src/main/java/Duke.java

```diff
@@ -1,10 +1,375 @@
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws IncorrectCommandInput {
+        Scanner sc = new Scanner(System.in);
+        LinkedList<Task> savedTasks = new LinkedList<Task>();
+        TaskList taskList = new TaskList(savedTasks);
+        checkExistingOutputFile(taskList);
+        printWelcomeMessage();
+        String userInput = "";
+        String commandInput = "";
+        String checkedCommandInput = "";
+        try {
+            userInput = sc.nextLine();
+            commandInput = identifyUserInput(userInput)[0];
+            checkedCommandInput = commandInputError(commandInput);
+        } catch (IncorrectCommandInput e){
+            System.out.println("Invalid Command!");
+        }
+        while(!userInput.contains("bye")){
+            switch(checkedCommandInput) {
+            case "todo":
+                try {
+                    addToDoTask(taskList, userInput);
+                    printTaskList(taskList);
+                } catch (StringIndexOutOfBoundsException e){
+                    System.out.println("Please include your description!");
+                }
+                break;
+
+            case "deadline":
+                try {
+                    addDeadlineTask(taskList, userInput);
+                    printTaskList(taskList);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("Please include your description!");
+                }
+                break;
+
+            case "event":
+                try {
+                    addEventTask(taskList, userInput);
+                    printTaskList(taskList);
+                } catch (StringIndexOutOfBoundsException e){
+                    System.out.println("Please include your description!");
+                }
+                break;
+
+            case "list":
+                listTask(taskList);
+                break;
+
+            case "done":
+                markTaskAsDone(taskList, userInput);
+                break;
+            case "delete":
+                try {
+                    deleteTaskFromList(taskList, userInput);
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("Please input a valid index.");
+                }
+                break;
+            default:
+                System.out.println("Invalid! Please try again.");
+            }
+            try {
+                userInput = sc.nextLine();
+                commandInput = identifyUserInput(userInput)[0];
+                checkedCommandInput = commandInputError(commandInput);
+            }catch (IncorrectCommandInput e){
```

Comment:
This is missing a character space between the `}` and `catch` statement. Refer to [Java coding standard > try-catch statement](https://se-education.org/guides/conventions/java/basic.html#layout).